src/server/index.tsx

```diff
@@ -46,6 +49,54 @@ server.get("/robots.txt", async (_req, res) => {
   res.send(robotstxt);
 });
 
+server.get("/css/themes-list", async (req, res) => {
+  const builtinThemes = [
+    "litera",
+    "materia",
+    "minty",
+    "solar",
+    "united",
+    "cyborg",
+    "darkly",
+    "journal",
+    "sketchy",
+    "vaporwave",
+    "vaporwave-dark",
+    "i386",
+    "litely",
+    "nord",
+  ];
+  fs.readdir(extraThemesFolder, function (_, data) {
+    if (data != null) {
+      data = data
+        .filter(d => d.endsWith(".min.css"))
+        .map(d => d.replace(".min.css", ""));
+      data = builtinThemes.concat(data);
+      // use set to remove duplicate values
+      data = Array.from(new Set(data));
+      res.send(data);
+    } else {
+      res.send(builtinThemes);
+    }
+  });
+});
+
+server.get("/css/themes/:name", async (req, res) => {
+  res.contentType("text/css");
+  const theme = req.params.name;
+  if (!theme.endsWith(".min.css")) {
+    res.send("Theme must be a css file");
+  }
+
+  const customTheme = path.resolve(`./${extraThemesFolder}/${theme}`);
+  if (fs.existsSync(customTheme)) {
+    res.sendFile(customTheme);
+  } else {
+    const internalTheme = path.resolve(`./dist/assets/css/themes/${theme}`);
```

Comment:
This will pry need to be `static/assets/css/themes`, like below.

Reply:
`server.use("/static", express.static(path.resolve("./dist")));`

So the http route `/static` is served from local path `./dist`.